eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+pr:
+  paths:
+    include:
+      - eng/pipelines/bicep-warnings.yml
```

Comment:
I assume you want to trigger this on other files as well?

Also you will need to list branches otherwise the paths will not take affect.

Reply:
Good point. Addressed in last two commits.